x/feegrant/exported/external.go

```diff
@@ -1,6 +1,8 @@
 package exported
 
 import (
```

Comment:
We don't use internal anymore. We should just have these defined under `x/feegrant/types`. Thoughts?

Reply:
Sure, it should be good to have it in feegrant/types.

Reply:
This will create a collision in names with FeeAllowance from proto, do you think we can let this be just like in evidence.
I moved the expected keepers to types though